src/system.h

```diff
@@ -82,8 +82,6 @@ class System {
     void send_heartbeat();
     void send_info_mqtt();
 
-    std::string getMacAddress() const;
```

Comment:
This causes link error,because it is refered by 
lib_deps = https://github.com/emsesp/EMS-ESP-Modules.git

Reply:
i was working on the code, but didn't update the repo. Just did now, so should work.

I'm do a workaround for the breaking CONFIG_IDF_TARGET_* and then merge